FastFollow.lua

```diff
@@ -260,130 +251,6 @@ windower.register_event('prerender', function()
-      return true
-    else
-      last_zone = os.clock()
-    end
-  elseif id == PACKET_OUT.ACTION and not casting then
-    if not pauseon:contains('spell') and not pauseon:contains('dismount') then return end
-    
-    local packet = packets.parse('outgoing', modified)
-    if packet.Category ~= PACKET_ACTION_CATEGORY.MAGIC_CAST and packet.CATEGORY ~= PACKET_ACTION_CATEGORY.DISMOUNT then return end
-    if packet.Category == PACKET_ACTION_CATEGORY.MAGIC_CAST and not pauseon:contains('spell') then return end
-    if packet.Category == PACKET_ACTION_CATEGORY.DISMOUNT and not pauseon:contains('dismount') then return end
-    
-    local cast_attempt = os.clock()
-    casting = cast_attempt
-    if pause_delay <= 0 then return end
-    
-    windower.ffxi.run(false)
-    running = false
-    coroutine.schedule(function()
-      packets.inject(packet)
-    end, pause_delay)
-    
-    local delay = pause_dismount_delay
-    if packet.Category == PACKET_ACTION_CATEGORY.MAGIC_CAST then
-      -- TODO: Maybe get a little smarter, such as checking if the target is within range, we have sufficient mp, etc.
-      local spell = spells[packet.Param]
-      delay = spell.cast_time + 0.5
-    end
-    
-    if co then coroutine.close(co) end
-    co = coroutine.schedule(function()
-      if casting and not (casting > cast_attempt) then
-        casting = false
-      end
-    end, pause_delay+0.5)
-    
-    return true
-  elseif id == PACKET_OUT.USE_ITEM and not casting then
-    if not pauseon:contains('item') then return end
-    
-    casting = os.time()
-    if pause_delay <= 0 then return end
-    
-    local packet = packets.parse('outgoing', modified)
-    
-    local item = items[packet.Param]
-    if not item or not item.cast_time then return end
-    
-    local cast_time = os.time()
-    casting = cast_time
-    
-    coroutine.schedule(function()
-      packets.inject(packets.parse('outgoing', modified))
-    end, pause_delay)
-    
-    if co then coroutine.close(co) end
-    co = coroutine.schedule(function()
-      if casting ~= cast_time then return end
-      casting = false
-    end, pause_delay+item.cast_time)
-    
-    return true
-  end
-end)
-
-windower.register_event('action', function(action)
-  local player = windower.ffxi.get_player()
-  if not player or action.actor_id ~= player.id then return end
-
-  if action.category == EVENT_ACTION_CATEGORY.SPELL_FINISH or (action.category == EVENT_ACTION_CATEGORY.SPELL_BEGIN_OR_INTERRUPT and action.param == EVENT_ACTION_PARAM.INTERRUPT) then
-    casting = false
-  elseif action.category == EVENT_ACTION_CATEGORY.ITEM_FINISH or (action.category == EVENT_ACTION_CATEGORY.ITEM_BEGIN_OR_INTERRUPT and action.param == EVENT_ACTION_PARAM.INTERRUPT) then
-    casting = false
-  elseif action.category == EVENT_ACTION_CATEGORY.SPELL_BEGIN_OR_INTERRUPT and action.param == EVENT_ACTION_PARAM.BEGIN then
-    casting = os.clock()
-  end
-end)
-
-function zone(zone_line, zone_type, zone, x, y)
```

Comment:
This is the function that does the packet injection. I suggest rejecting this PR and limit our changes to this function instead of deleting everything concerning packets.